src/Common/Commands.ScenarioTests.Common/Common.ps1

```diff
@@ -280,7 +280,9 @@ function Wait-Function
 
     do
     {
-        Start-Sleep -s 5
+        if ($env:AZURE_TEST_MODE -eq "Record"){
+          Start-Sleep -s 5
```

Comment:
There is a common sleep function that does exactly this:  https://github.com/Azure/azure-powershell/blob/dev/src/Common/Commands.ScenarioTests.Common/Common.ps1#L306